terraform/environments/mlra/platform_secrets.tf

```diff
@@ -1,33 +1,33 @@
 ######################### Run Terraform via CICD ##################################
```

Comment:
This will break the pipeline as is - can you revert commenting in these file

Reply:
I will remove the comments once I'm ready to run the code in CICD